Source/r_sky.c

```diff
@@ -32,6 +32,10 @@
 
 #include "r_sky.h"
 #include "r_state.h" // [FG] textureheight[]
+#include "r_data.h"
+#include "w_wad.h"
+
+extern int *texturewidth;
```

Comment:
I included `r_state.h` only for `textureheight[]`, so if you like please  add `texturewidth[]` to `r_state.h`.

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/483/commits/ee455c291efa9afb2d6fb3b5e3f757e87a001de5